src/be_txn.erl

```diff
@@ -214,4 +238,34 @@ to_json(blockchain_txn_payment_v2, T, _Ledger) ->
       <<"payments">> => [PaymentJson(Payment) || Payment <- blockchain_txn_payment_v2:payments(T)],
       <<"fee" >> => blockchain_txn_payment_v2:fee(T),
       <<"nonce">> => blockchain_txn_payment_v2:nonce(T),
-      <<"signature">> => ?BIN_TO_B64(blockchain_txn_payment_v2:signature(T)) }.
+      <<"signature">> => ?BIN_TO_B64(blockchain_txn_payment_v2:signature(T)) };
+to_json(blockchain_txn_state_channel_open_v1, T, _Ledger) ->
+    #{<<"id">> => ?BIN_TO_B64(blockchain_txn_state_channel_open_v1:id(T)),
+      <<"owner">> => ?BIN_TO_B58(blockchain_txn_state_channel_open_v1:owner(T)),
+      <<"oui">> => blockchain_txn_state_channel_open_v1:oui(T),
+      <<"fee" >> => blockchain_txn_state_channel_open_v1:fee(T),
+      <<"nonce">> => blockchain_txn_state_channel_open_v1:nonce(T),
+      <<"expire_within">> => blockchain_txn_state_channel_open_v1:expire_within(T),
+      <<"signature">> => ?BIN_TO_B64(blockchain_txn_state_channel_open_v1:signature(T)) };
+to_json(blockchain_txn_state_channel_close_v1, T, _Ledger) ->
+
+    SummaryJson = fun(Summary) ->
+                          #{ <<"client_pubkeybin">> => ?BIN_TO_B58(blockchain_state_channel_summary_v1:client_pubkeybin(Summary)),
+                             <<"num_dcs">> => blockchain_state_channel_summary_v1:num_dcs(Summary),
+                             <<"num_packets">> => blockchain_state_channel_summary_v1:num_packets(Summary)
+                           }
+                  end,
+
+    SCJson = fun(SC) ->
+                     #{<<"id">> => ?BIN_TO_B64(blockchain_state_channel_v1:id(SC)),
+                       <<"owner">> => ?BIN_TO_B58(blockchain_state_channel_v1:owner(SC)),
+                       <<"nonce">> => blockchain_state_channel_v1:nonce(SC),
+                       <<"summaries">> => [SummaryJson(B) || B <- blockchain_state_channel_v1:summaries(SC)],
+                       <<"root_hash">> => ?BIN_TO_B64(blockchain_state_channel_v1:root_hash(SC)),
+                       <<"state">> => blockchain_state_channel_v1:state(SC),
```

Comment:
Can you actually have strings for the states here? like `<<"open">>` and `<<''closed">>`? It'll make queries easier